requests/util.js

```diff
@@ -43,6 +43,154 @@ function getOooQueryParamsString(query) {
   }
   return `?${queryParam}`;
 }
+function addEmptyPageMessage(container) {
+  const emptyPageMessage = createElement({
+    type: 'p',
+    attributes: { class: 'page-message' },
+    innerText: 'No extension requests to show!',
+  });
+  container.appendChild(emptyPageMessage);
+}
+
+async function getAllUsersStatus() {
+  try {
+    const res = await fetch(`${API_BASE_URL}/users/status`, {
+      credentials: 'include',
+      method: 'GET',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+    return await res.json();
+  } catch (error) {
+    console.error(error);
+  }
+}
+
+async function updateExtensionRequestStatus({ id, body }) {
+  const url = `${API_BASE_URL}/extension-requests/${id}/status`;
+  const res = await fetch(url, {
+    credentials: 'include',
+    method: 'PATCH',
+    body: JSON.stringify(body),
+    headers: {
+      'Content-type': 'application/json',
+    },
+  });
```

Comment:
We have the `/requests` API available for all types of requests. Please use this API and refer to the API contract documentation for any related information.